dts/bindings/sdhc/sdhc.yaml

```diff
@@ -59,3 +59,13 @@ properties:
     type: boolean
     description: |
       The host controller supports HS400 at 1.8V
+
+  mmc-dw-8bit:
```

Comment:
This commit needs to be first in the PR, since the emmc driver depends on these properties.

Also, please use a title similar to the following:

`dts: bindings: sdhc: add dts entry for sdhc bus width`

Reply:
This commit is before driver commit.